Dockerfile.linux.imagebuilder

```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20180117125404
```

Comment:
I think a usage note should be added similar to the one Dockerfile.linux.testrunner has.

Reply:
Ok. How about -

```
# Use this Dockerfile to create an image containing this repo and ImageBuilder
# Usage: docker build -t imagebuilder -f ./Dockerfile.linux.imagebuilder .
```